packages/easy-hydrator/src/ArrayToValueObjectHydrator.php

```diff
@@ -41,12 +38,10 @@ public function hydrateArray(array $data, string $class): object
         /** @var CacheItem $cacheItem */
         $cacheItem = $this->filesystemAdapter->getItem($arrayHash);
         if ($cacheItem->get() !== null) {
-            return $cacheItem->get();
+            // return $cacheItem->get();
         }
 
-        $arguments = $this->resolveClassConstructorValues($class, $data);
-
-        $value = new $class(...$arguments);
+        $value = $this->objectCreator->create($class, $data);
```

Comment:
More standard name is `*Factory`
```suggestion
        $value = $this->objectFactory->create($class, $data);
```